org.adempiere.base/src/org/compiere/model/I_AD_Field.java

```diff
@@ -75,6 +75,28 @@ public interface I_AD_Field
 
 	public org.compiere.model.I_AD_Column getAD_Column() throws RuntimeException;
 
+    /** Column name AD_Field_ID */
```

Comment:
is it possible that you are using a slightly different version of the core ModelGenerator classes?

why the order of the fields are different when you run it?

or is maybe the collation setting on your postgres?

Reply:
@nmicoud - what is the result of running the following query in your database?
```sql
SELECT 'AD_Field_UU' UNION
SELECT 'AD_Field_ID' UNION 
SELECT 'AD_Column_ID' UNION
SELECT 'AD_FieldGroup_ID'
ORDER BY 1
```

in my database the fields are ordered this way:
AD_Column_ID
AD_FieldGroup_ID
AD_Field_ID
AD_Field_UU

Reply:
nevermind - I found the issue and opened ticket and pull request:
https://idempiere.atlassian.net/browse/IDEMPIERE-6149